spec/predicates/reference.md

```diff
@@ -0,0 +1,143 @@
+# Predicate type: Reference
+
+Type URI: https://in-toto.io/attestation/reference/v0.1
+
+Version: v0.1.0
+
+## Purpose
+
+The reference attestation references a set of documents that are relevant to
+some resource. It is used to point to and sign metadata that is not
+provided in-band.
+
+## Use Cases
+
+One use case is to provide an SBOM of an artifact without needing to sign
+the entire SBOM and transmit it in-band. A consumer, such as
+[GUAC](https://guac.sh/), retrieves the SBOM, verifies the digest, and uses it
+for vulnerability management, compliance, analysis, or any other purpose.
+
+Another use case is to provide evidence for a policy such as "deny unless a
+reference to an SBOM is provided".
+
+There is encodability overlap with some other predicates, mainly SCAI, but
+the use case is distinct.
+
+## Prerequisites
+The
+[in-toto Attestation Framework](https://github.com/in-toto/attestation/blob/main/spec/README.md)
+and an SBOM specification such as [SPDX](https://spdx.dev/).
+
+## Model
+This predicate is intended to be generated and consumed throughout the software
+supply chain. In addition, it is intended to be used in the analysis of it as a
+whole.
+
+## Schema
+
+```jsonc
+{
+  // Standard attestation fields:
+  "_type": "https://in-toto.io/Statement/v1",
+  "subject": [{ ... }],
+
+  // Predicate:
+  "predicateType": "https://in-toto.io/attestation/reference/v0.1",
+  "predicate": {
+    "attester": {
+      "id": "<TypeUri>"
+    },
+    "references": ["<ResourceDescriptor>", ...]
+  }
+}
+```
+
+### Parsing Rules
+This predicate follows the
+[in-toto Attestation Framework's parsing rules](../v1/README.md#parsing-rules).
+
+The `references` field is repeated as an optimization: multiple references
+SHOULD be treated as if each reference was represented in its own attestation
+with all other content unchanged.
+
+The reference(s) are associated with the set of subjects as a whole, instead of
+being independently associated with each subject. See the
+[example](#reference-to-an-sbom-for-multiple-artifacts) with two subjects.
+
+### Fields
+`attester.id`: string ([TypeUri](../v1/field_types.md#typeuri)), *required*
+
+An identifier for the system that provides the document.
+
+`references` array of [ResourceDescriptor](../v1/field_types.md#resource_descriptor) objects, *required*
+
+The referred documents. The `mediaType` and `downloadLocation` of each SHOULD
```

Comment:
Should `downloadLocation` be "MUST"?  How useful would a reference attestation that doesn't point to something be?

Reply:
Makes sense to me, changed to MUST.

Reply:
I think if the the `uri` is left empty and `digest` is provided, then wouldn't be a conflict.